### PR TITLE
Fixed issues 2496. Used option 4: parse_version.cmake to avoid python3 usage.

### PR DESCRIPTION
**Pull request by proydakov**
Hi, GLSLANG Team.

I implemented fix to project version detection from CHANGED.md without python with approach 4.
https://github.com/KhronosGroup/glslang/issues/2496

Could you review and accept my changes, pls?

Best regards, Proydakov Evgeny.

### CODE REVIEW

File: parse_version.cmake [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# Copyright 2020 The Marl Authors.
```
Comment (ben-clayton) [inline parse_version.cmake:1]: As the sole author of this cmake file from marl, you have my permission to change this license to the standard 3-Clause BSD License used by glslang. Please change the first line to `Copyright (C) 2020 Google, Inc.`

Comment (proydakov) [reply parse_version.cmake]: Done. Thx, Ben

**Comment by ben-clayton** [comment]
LGTM.
Glslang maintainers: Please can you apply the kokoro-run label so we can verify this against the various build systems?

**Comment by greg-lunarg** [comment]
Given Ben's review and a successful kokoro run, merging.

**Comment by greg-lunarg** [comment]
> Given Ben's review and a successful kokoro run, merging.

Sorry! Cancel that. I thought the kokoro had completed, but then saw the failure for ubuntu-clang-gn.

@proydakov Can you please take a look?

**Comment by ben-clayton** [comment]
I can't seem to find a log for that failed build, so maybe it was infra failure. Attempting to retry it.

**Comment by ben-clayton** [comment]
The `CI-ubuntu-clang-gn` build issue is because the PR branch is considerably behind master, and is missing the kokoro build configuration file at `glslang/kokoro/linux-clang-gn/presubmit.cfg` 
I'd suggest rebasing the PR branch. The `kokoro:run` label will need re-applying after the rebase.

**Comment by proydakov** [comment]
kokoro:run

**Comment by proydakov** [comment]
+kokoro:run

**Comment by proydakov** [comment]
#kokoro:run

**Comment by proydakov** [comment]
@ben-clayton I rebased pr branch. Could you add label for PR, pls ?

**Comment by ben-clayton** [comment]
I'm not a maintainer for glslang. @greg-lunarg - please can you do the honours?

**Comment by greg-lunarg** [comment]
All kokoro passing. With Ben's review, merging.

**Comment by greg-lunarg** [comment]
Thanks @proydakov and @ben-clayton !

**Pull request merged**